client/e2e/workflows/bceidbusiness/industry_user.spec.ts

```diff
@@ -0,0 +1,222 @@
+// ℹ️ Environment variables
+import * as dotenv from "dotenv";
+import { OperatorPOM } from "@/e2e/poms/operator";
+import { fieldsUpdate } from "@/e2e/utils/helpers";
+import { deleteUserOperatorRecord } from "@/e2e/utils/queries";
+dotenv.config({ path: "./e2e/.env.local" });
+const happoPlaywright = require("happo-playwright");
+
+test.beforeEach(async ({ context }) => {
+  await happoPlaywright.init(context);
+  let response: APIResponse = await context.request.get(
+    "http://localhost:8000/api/registration/test-setup",
+  );
+  // Wait for the response and check for success status text and code (e.g., 200)
+  expect(await response.text()).toBe("Test setup complete.");
+  expect(response.status()).toBe(200);
+
+  await deleteUserOperatorRecord(process.env.E2E_INDUSTRY_USER_GUID as string);
+});
+
+test.afterEach(async () => {
+  await happoPlaywright.finish();
+});
+
+// 🏷 Annotate test suite as serial
+test.describe.configure({ mode: "serial" });
+test.describe("Test Workflow industry_user", () => {
+  // 👤 run test using the storageState for this role
+  const storageState = process.env.E2E_INDUSTRY_USER_STORAGE;
+  // Note: specify storageState for each test file or test group, instead of setting it in the config. https://playwright.dev/docs/next/auth#reuse-signed-in-state
+  test.use({ storageState: storageState }); // this will error if no such file or directory
+
+  test("Test Redirect to Dashboard", async ({ page }) => {
+    // 🛸 Navigate to dashboard page
+    const dashboardPage = new DashboardPOM(page);
+    await dashboardPage.route();
+    // 🔍 Assert that the current URL ends with "(authenticated)/dashboard"
+    await dashboardPage.urlIsCorrect();
+    const pageContent = page.locator("html");
+    await happoPlaywright.screenshot(dashboardPage.page, pageContent, {
+      component: "Industry User Dashboard page",
+      variant: "industry_user",
+    });
+  });
+
+  test("Select existing operator (via legal name) and request admin access", async ({
+    page,
+  }) => {
+    let pageContent;
+    // 🛸 Navigate to dashboard page
+    const dashboardPage = new DashboardPOM(page);
+    const selectOperatorPage = new OperatorPOM(page);
+    await dashboardPage.route();
+    // 🔍 Assert that the current URL ends with "(authenticated)/dashboard"
+    await dashboardPage.urlIsCorrect();
+    // brianna you need to clear the user_operator table for this, Sepehr is working on it
+    await dashboardPage.clickSelectOperatorTile();
+    await selectOperatorPage.urlIsCorrect();
+
+    pageContent = page.locator("html");
+    await happoPlaywright.screenshot(page, pageContent, {
+      component: "Select operator page",
+      variant: "default",
+    });
+
+    await page.getByPlaceholder("Enter Business Legal Name").click();
+    await page.getByPlaceholder("Enter Business Legal Name").fill("Operator");
+    await page.getByText(/Operator 1 Legal Name/i).click();
+    // happo--debatable here
```

Comment:
There's only one form field on this page so I decided not to add a filled happo shot here but happy to be convinced